src/resources/js/Pages/Welcome.vue

```diff
@@ -178,7 +195,7 @@ export default {
                 markers: [
                 ],
             },
-            search_q: '43210',
+            search_q: 'Zipcode',
```

Comment:
I don't think we want to populate this value at all initially. I'll remove this default value.